pages/index.js

```diff
@@ -1,8 +1,56 @@
 import styled from "styled-components";
 import SpotCard from "../components/SpotCard";
-import spots from "../helpers/mockSpots";
+import CreateSpot from "../components/CreateSpot";
+import { Button } from "../components/Button";
+import { useState, useEffect } from "react";
 
 export default function Home() {
+  const [state, setState] = useState("hide");
+  const [spots, setSpot] = useState(() => {
+    try {
+      return JSON.parse(localStorage.getItem("spotsList") ?? []);
+    } catch (error) {
+      console.warn(error);
+      return [];
+    }
+  });
+  useEffect(() => {
+    localStorage.setItem("spotsList", JSON.stringify(spots));
+  }, [spots]);
```

Comment:
I would highly recommend using the useLocalStorage hook we discussed on Friday. This has some advantages, for example the reusability ♻️